packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -166,6 +166,24 @@ export default /* @ngInject */ ($stateProvider) => {
           servers: [server],
         });
       },
+
+      vpsStateInfo: /* @ngInject */ (stateVps) => {
```

Comment:
Not now but in the future, we can move this to vps.class and have methods like isError(), isSuccess(), and isWarning() etc.

Reply:
True, i thought like you.
There are a lot work to complete.
I will create a Jira to be sure we do it.